core/templates/.spacemacs.template

```diff
@@ -79,6 +79,8 @@ values."
    ;; List of items to show in the startup buffer. If nil it is disabled.
    ;; Possible values are: `recents' `bookmarks' `projects'.
    ;; (default '(recents projects))
+   dotspacemacs-version-check-enable nil
```

Comment:
This probably should go down below the line that starts `dotspacemacs-startup-lists`, since you placed it between the comment and the variable it describes.  Also, the comment describing the new variable should go before it.